packages/zipkin/src/tracer/index.js

```diff
@@ -21,6 +21,8 @@ class Tracer {
     this.sampler = sampler;
     this._ctxImpl = ctxImpl;
     this._defaultTraceId = this.createRootId();
+    this._startTimestamp = now();
+    this._startTick = process && process.hrtime ? process.hrtime() : undefined;
```

Comment:
oops.. @sveisvei this is why I used ternary, as I had it defined twice and seemed quicker as a one-line paste. should I define getHrTime again? or maybe add it to time.js?